playwright/e2e/crypto/utils.ts

```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { type Page, expect, JSHandle } from "@playwright/test";
+import { expect, JSHandle, type Page } from "@playwright/test";
 
-import type { CryptoEvent, ICreateRoomOpts, MatrixClient } from "matrix-js-sdk/src/matrix";
+import type { ICreateRoomOpts, MatrixClient } from "matrix-js-sdk/src/matrix";
```

Comment:
`CryptoEvent` is still referenced below, so this needs to stay, I think.

Note that it's important it be `import type` rather than `import` (since playwright tests aren't allowed to have runtime dependencies on the js-sdk).